course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,103 @@
+
+# Recommended resources
+- [R for Data science](http://r4ds.had.co.nz/), H Wickham, G Grolemund, 2017
+    - Excellent open access resource for R.
+- [Rstudio cheat sheets](https://www.rstudio.com/resources/cheatsheets/), Rstudio, 2017
+    - As good as it sounds, great quick reference.
+
+# Assignments / Course learning outcomes
+The main goals are to:
+1) integrate and apply concepts and theoretical models learnt in class,
+2) learn how to collect data with real organisms (technical skills), 
+3) learn how to use basic statistics to analyze and interpret data,
+4) realize that data do not always fit theoretical models, but that it is important to interpret data with a larger theoretical framework in mind, 
+5) learn how to work in group. 
+
+The due dates for each lab report are listed on the lab schedule. Hand in a hard copy of your lab reports at the beginning of the lab. 
+
+# Improving your writing skills
+Effective communication is crucial in science. The University of Toronto provides services to help you improve your writing (see specific section on lab reports), from general advices on effective writing to writing centers and writing courses. See http://www.writing.utoronto.ca/. The Faculty of Arts & Science also offers an English Language Learning (ELL) program, which provides free individualized instruction in English skills. Take advantage of these!
+
+# Academic integrity:
+You should be aware of the University of Toronto Code of Behaviour on Academic Matters. Also see http://www.writing.utoronto.ca/advice/using-sources/how-not-to-plagiarize on How Not to Plagiarize. Note that it is NOT appropriate to use large sections from internet sources, and inserting a few words here and there does not make it an original piece of writing. Be careful in using internet sources – there is no review of most online material and there are MANY errors out there. Use only academic or government internet sources when absolutely necessary. Make sure you read material from many sources (published, peer-reviewed, trusted internet sources) and that you write an original text using this information. Always cite your sources. In case of doubt about plagiarism, talk to your instructor.
+
+# Lecture schedule
+| Week | Date   | Topic                                               | Instructor     |
+|------|--------|-----------------------------------------------------|----------------|
+| 1    | Sep 07 | Welcome to the course! Intro to programming         | Everyone       |
+| 2    | Sep 12 | Basic R – vectors, data frames                      | Joel           |
+| 3    | Sep 14 | Basic R – scripting, conditionals, loops            | Joel           |
+| 4    | Sep 19 | Intro to dplyr                                      | Joel           |
+| 5    | Sep 21 | More dplyr                                          | Joel           |
```

Comment:
Similar to my comment about Rmarkdown, I would like this lecture to include very basic plotting with ggplot after some dplyr wrangling. I think this could also be helpful for the students to know before @mbonsma 's lectures. @lcoome 's lectures will then go into much more depth and breadth of ggplot and use them practically on the students' own projects.